Uploader/Properties/AssemblyInfo.cs

```diff
@@ -1,36 +1,26 @@
-﻿using System.Reflection;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
-
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Uploader")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("Uploader")]
-[assembly: AssemblyCopyright("Copyright ©  2018")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("a01a7519-e10d-4153-8cd4-678935721df3")]
-
-// Version information for an assembly consists of the following four values:
+﻿//------------------------------------------------------------------------------
+// <copyright company="CQSE GmbH">
+//   Copyright (c) CQSE GmbH
+// </copyright>
+// <license>
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
 //
-//      Major Version
-//      Minor Version
-//      Build Number
-//      Revision
+//      http://www.apache.org/licenses/LICENSE-2.0
 //
-// You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+// </license>
+//------------------------------------------------------------------------------
+
+using System.Reflection;
+
+[assembly: AssemblyProduct("CQSE .NET Profiler Uploader")]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
Also maybe a minor thing, but we could patch the assembly version according to: https://www.appveyor.com/docs/build-configuration/#assemblyinfo-patching so it reflects the build version.